android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -531,6 +611,47 @@ class XMTPModule : Module() {
             ConversationWrapper.encode(client, conversation)
         }
 
+        AsyncFunction("createGroup") { clientAddress: String, peerAddresses: List<String> ->
+            logV("createGroup")
+            val client = clients[clientAddress] ?: throw XMTPException("No client")
+            if (client.libXMTPClient == null) {
+                throw XMTPException("Create client with enableAlphaMLS true in order to create a group")
+            }
+            val group = client.conversations.newGroup(peerAddresses)
+            logV("id after creating group: " + Base64.encodeToString(group.id, NO_WRAP))
+            val encodedGroup = GroupWrapper.encode(client, group)
+            return@AsyncFunction encodedGroup
+        }
+
+        AsyncFunction("listMembers") { clientAddress: String, groupId: String ->
```

Comment:
I might change this to `listMemberAddresses` in case we do create some sort of member object down the road

Reply:
fixed here 19e03d84fa660046ebb6dae777fe3811f2b22868